engine/index.js

```diff
@@ -298,6 +299,107 @@ function populateCanIUsePercent(canIUseData, features) {
   });
 }
 
+const statusFields = ['firefox_status', 'spec_status', 'opera_status',
+                      'webkit_status', 'ie_status'];
+// checking for changes in 'status' object
+function checkForNewData(features, dbTestNumber) {
+  return redis.getClient(dbTestNumber)
+  .then((client) => new Promise((resolve, reject) => {
+    client.get('status', (err, oldStatus) => {
+      try {
+        oldStatus = JSON.parse(oldStatus);
+      } catch (e) {
+        console.log(e);
+      }
+      if (err) {
```

Comment:
Should this block be above the `try` block? It seems like we don't want to try to parse `oldStatus` if our call to `client.get` gave us back an error